### PR TITLE
Doc about installation and readme contributing

### PR DESCRIPTION
**Pull request by umpirsky**

### CODE REVIEW

File: README.tpl [hunk 1]
```diff
@@ -2,6 +2,52 @@
 
 Analyzes your Symfony project and tries to make it compatible with the new version of Symfony framework.
 
+## Installation
+
+### Manual
+
+#### Local
+
+Download the ``symfony-upgrade-fixer.phar`` file and store it somewhere on your computer.
```
Comment (akovalyov) [inline README.tpl]: Maybe it is worth it to mention that one should take it from releases page?

File: README.tpl [hunk 2]
```diff
@@ -2,6 +2,52 @@
...
+your system:
+
+```bash
+$ wget https://github.com/umpirsky/Symfony-Upgrade-Fixer/releases/download/v0.1.0/symfony-upgrade-fixer.phar -O symfony-upgrade-fixer
+```
+then:
+
+```bash
+$ sudo chmod a+x symfony-upgrade-fixer
+$ sudo mv symfony-upgrade-fixer /usr/local/bin/symfony-upgrade-fixer
```
Comment (akovalyov) [inline README.tpl]: Proposal:
```bash
$ sudo wget https://github.com/umpirsky/Symfony-Upgrade-Fixer/releases/download/v0.1.0/symfony-upgrade-fixer.phar -O /usr/local/bin/symfony-upgrade-fixer 
$ sudo chmod a+x /usr/local/bin/symfony-upgrade-fixer
```
One line less :)

**Comment by akovalyov** [comment]
@umpirsky I thought about putting the link, like  to the `phar` file to README, but well, it is trouble since Github doesn't provide the direct link to download file in release. So you will have to update the doc each time you have a new release. 
So, it's time to think about the website :)

**Comment by umpirsky** [comment]
@akovalyov Я коварен https://github.com/umpirsky/Symfony-Upgrade-Fixer/pull/12/files#diff-04ef6d8a9b042b31c46af90dfe211466R30 :)

**Pull request merged**